Sources/Sentry/SentryTracer.m

```diff
@@ -140,24 +141,30 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
         // Store current amount of frames at the beginning to be able to calculate the amount of
         // frames at the end of the transaction.
         SentryFramesTracker *framesTracker = [SentryFramesTracker sharedInstance];
+#    if SENTRY_TARGET_PROFILING_SUPPORTED
+        framesTracker.currentTracer = self;
+#    endif // SENTRY_TARGET_PROFILING_SUPPORTED
```

Comment:
`h`: This should only be called if `[_hub getClient].options.enableProfiling` is enabled. Otherwise, we track the timestamps even if profiling is not enabled.